Sources/Extensions/HyperwalletFee.swift

```diff
@@ -44,29 +44,28 @@ extension HyperwalletFee {
         case percent(_ percentFee: HyperwalletFee)
         case mixed(_ percentFee: HyperwalletFee, _ flatFee: HyperwalletFee)
 
-        func feeDescription() -> String {
+        func feeDescription(for currency: String) -> String {
             switch self {
             case .flat(let flatFee):
-                return flatFeeDescription(flatFee)
+                return flatFeeDescription(flatFee, currency)
 
             case .percent(let percentFee):
-                return percentFeeDescription(percentFee)
+                return percentFeeDescription(percentFee, currency)
 
             case let .mixed(percentFee, flatFee):
-                return mixedFeeDescription(percentFee, flatFee)
+                return mixedFeeDescription(percentFee, flatFee, currency)
             }
         }
 
-        private func flatFeeDescription(_ flatFee: HyperwalletFee) -> String {
+        private func flatFeeDescription(_ flatFee: HyperwalletFee, _ currency: String) -> String {
             let feeFormat = "fee_flat_formatter".localized()
-            return String(format: feeFormat.localized(), flatFee.currency, flatFee.value)
+            return String(format: feeFormat.localized(), currency, flatFee.value)
         }
 
-        private func percentFeeDescription(_ percentFee: HyperwalletFee) -> String {
+        private func percentFeeDescription(_ percentFee: HyperwalletFee, _ currency: String) -> String {
```

Comment:
Hi Yurii, we missed one use case for this method. Could you please add this code at the end of if/else 

```
else {
                feeFormat = "fee_percent_no_min_and_max_formatter".localized()
                description = String(format: feeFormat, value)
            }
```

Reply:
And also add ```"fee_percent_no_min_and_max_formatter" = "%1$@%%";``` to  Localizable.strings. Thank you.

Reply:
sure, fixed